.github/CONTRIBUTING.md

```diff
@@ -94,11 +94,20 @@ Afterwards, to clean up:
 
 1. In the directory of your stencil project, run `npm install --save-dev stencil@<VERSION>` for the `<VERSION>` of Stencil core that was installed in your project prior to testing. 
 
-## Commit Message Format
+### Commit Message Format
 
-We have very precise rules over how our git commit messages should be formatted. This leads to readable messages that are easy to follow when looking through the project history. We also use the git commit messages to generate our changelog. (Ok you got us, it's basically Angular's commit message format).
+We strive to adhere to a consistent commit message format of:
```

Comment:
Should we officially embrace [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)?

Reply:
I strayed away here from referencing conventional commits, because they mark `scope` as optional. Turns out we don't use the scope explicitly when [we generate a changelog](https://github.com/ionic-team/stencil/blob/42cdeaec424fb053648a5ae97e611a7c58d69788/scripts/utils/release-utils.ts#L56) like I thought we did. Let me dig into that conventional commit package a little bit more - when I did the 2.7.0-0 dev release, some things didn't get populated in the `CHANGELOG.md` file in the root of this repo the way I thought they would

Reply:
@ltm I spent some time experimenting with our changelog generation scripts, and made a few updates to tailor the advice to the current behavior of the package we use.

I did update the verbiage to state we use the _Angular_ version of conventional commits, which is a little different in that:
- It does not use the '!' syntax for breaking changes (commits with this syntax won't get added to our `CHANGELOG` under the BREAKING CHANGES section today, this is a result of how the tooling is configured ATM)
- I added instructions on using the 'BREAKING CHANGE:' footer syntax. This _does_ get picked up in the BREAKING CHANGES section of the CHANGELOG
- I added an example that hopefully paints a picture of how a commit translates to entries on `CONTRIBUTING.md`

I also added `revert` to the acceptable types list, stealing that from the Angular folks as well.